src/test/resources/solr/collection1/conf/schema.xml

```diff
@@ -17,5 +17,6 @@
 
 	<field name="title"     type="string" indexed="true"  stored="true" />
 	<field name="access"    type="string" indexed="false" stored="true" default="c:0" docValues="true" />
+	<field name="accesses"  type="string" indexed="false" stored="true" default="c:0" docValues="true" multiValued="true" />
```

Comment:
Why do you want to add another field? I would just make the `access` field multi value.